### DISCUSSION TITLE
پیشنهاد ساخت و طراحی اپلیکیشن و نرم افزار مختص کاربران ایرانی

### DISCUSSION THREAD

**Discussion by Phoenix-999** [iranxray/hope#95] [Ideas]
با سلام به همه برو بچه ها @iranxray
 اول از همه برای پست کردن این موضوع تکراری عذرخواهی میکنم

همانطوری که خیلی از بچه های  قدیمی و پیشکسوت در این مورد خاص اقداماتی انجام داده اند و تلاشهای هم صورت گرفته در زمینه آموزش و بهبود وضیت موجود برای راحتی استفاده کاربران مبتدی ایرانی که از اپلیکیشن‌ها و نرم افزارهای موجود در بازار که قابلیت دریافت کانفیگ های V2ray & Xray را دارند, بتوانند استفاده بهینه کنند. 
اما با تمام این تفاسیر, به علت تغییر روشهای فیلترینگ و خودآموزی روزانه AI GFW  بیشتر راه حل ها و روشها اتصال منسوخ شده یا فاقد کارایی هست, این حلقه هر روز در حال تنگ تر شدن هست و این بازی موش و گربه داره به نفع گربه خبیث تمام میشه.

تجربیات کاربران نشون میده که خیلی از این اپلیکیشن ها و تولید کنندگان آنها با سرعت و دقتی  که ما انتظار و به آنها نیاز داریم, مشکلات را  بر طرف نمیکنند و راه حل های جدید را برای اتصال کاربران ایرانی بر روی اپلیکیشن ها یا نرم افزارهای خود به روز رسانی نمیکنند.
به عنوان مثال @radio-azad-iran هر روز در خط مقدم داره برای حل این مشکلات با تولیدکنندگان اپلیکیشنهای مخلتف با چنگ و دندان میجنگه, ولی با این حال ما داریم زمان را از دست میدهم, با این که بیشتر سازندگان با ما در ارتباط نزدیک بودند و تا آنجایی که برایشان مقدور بوده همکاری با جامه ایرانی داشتند اما ایرانی تبار نیستند و الویت آنها رفع مشکل کاربران ایرانی نیست, به به قول معروف  **`They have NO dog in this fight`**

من فکر میکنم همه ما مشکل را دقیقا میدانیم و درک میکنیم پس توضیح بیشتر نیاز نیست
فکر کنم وقت آن رسیده که تا دیر نشده, دست به کار ساخت و طراحی اپلیکیشن و نرم افزار بر روی همه پلاتفرم ها که مختص کاربران ایرانی باشد, شویم تا در این جنگ نابرابر مغلوب نشویم.

دوستانی هستند که آماده همکاری مالی و اطلاعتی را در این زمینه دارند, ولی برای شروع این پروژه نیاز به تشکیل یک تیم خبره و کارآمد که کاملا به مشکلات کاربران ایرانی آگاهی داشته باشدند داریم. 
مطمئن هستم اگه این پروژه کلیک بخورد همه به ما ملحق خواهند شد به قول بچه های @iranxray 
**`دست‌های هم رو بگیریم، تنهایی از این جا نمی‌تونیم گذر کنیم.`**

**Comment by sambali9** [comment]
در حال حاظر مشکل اصلی اینه که برای IOS نیاز به ساخت یک هسته بسیار بهینه داریم که روی IOS های پایین جواب بده. ما همینطور نیاز به متخصصانی داریم که باید از کارشون بزنن تا کمک کنن. من کاری که تو ذهنمه اینه که یک برنامه برای تمامی پلتفرم ها داشته باشیم و همینطور یک هسته اختصاصی که هم از قابلیت های xray برخوردار باشه و همینطور از قابلیت های مخصوص اینترنت ایران یک مثالش همون فرگمنت هستش که شاید فقط برای ایران به درد بخوره.
این چندتا نکته توی ذهن منه حالا افراد میتونن هر مشکلی دیدن درش رو بگن:
۱. هسته فوق بهینه که هم از قابلیت های xray/v2ray پشتیبانی کنه هم از قابلیت های مخصوص اینترنت ایران. ترجیها با زبان هایی که نیاز به رم کمی دارن نوشته شده باشه مثل rust, c, c++.
۲. یک رابط کاربری همه گانی تریجها با فلاتر یا ری اکت نیتیو نوشته شده باشه که همه پلتفرم هارو بتونه ساپورت کنه.
۳. پروژه باید کاملا متن باز باشه و هیچ قسمتیش نباید مخفی باشه

مهمتر از همه اینه که پروژه استارت بخوره چون اگه فقط اعلامیه بزنیم به کسی کمکی نمیکنه ترجیها گروه تلگرامی ساخته بشه تا همه افراد سریعا با هم در ارتباط باشن. ما نیاز به کسی داریم که مسولیت مدریت اینکارو بپذیره و همه رو بتونه ۲۴ ساعته مدیریت کنه که واقعا کار سختی هستش امیدوارم دوستان علاقه مند همینجا اعلام امادگی کنند. هرچه سریعتر کار شروع بشه بهتره.
